thermalnetwork/network.py

```diff
@@ -600,7 +610,9 @@ def run_sizer_from_cli_worker(
     # begin populating structures in preparation for sizing
     errors = 0
     errors += network.set_design(des_method_str=ghe_design_data["method"], throw=True)
-    errors += network.set_components(network_data, throw=True)
+    errors += network.set_components(
+        network_data, system_parameters_data["district_system"]["fifth_generation"], throw=True
+    )
```

Comment:
Is it reasonable to only pass in the district params? Would we want building/ETS params for any reason?